src/libmongoc/tests/test-libmongoc.c

```diff
@@ -1649,6 +1649,78 @@ test_framework_client_new ()
    return client;
 }
 
+mongoc_server_api_t *
+test_framework_get_default_server_api (void)
+{
+   char *require_api_version = test_framework_getenv ("MONGODB_API_VERSION");
```

Comment:
[nit] Maybe just `api_version`? `require_api_version` implies to me that this would be a `bool`.

Reply:
Done